app/cypress/integration/index.spec.js

```diff
@@ -468,11 +468,6 @@ context('Homepage', () => {
 
     cy.get('h1').contains('Submission');
 
-    cy.get('input[id="root_submissionCompletedFor"]').type('test');
-
-    cy.get('#root_submissionDate').click();
-    cy.get(`.react-datepicker__day--001`).eq(2).click({ force: true });
-
```

Comment:
You previously told me that this end to end test was supposed to follow the happy path and didn't need to test every input?

Reply:
I was debating commenting on this one, but I think there's a difference between reducing our test coverage vs adding perfect tests: while changes don't need to have perfect coverage, they shouldn't reduce existing tests coverage

In particular here, since the feature relies on something that happens on another page (inserting the `submissionCompletedFor` and `submissionDate` when the other forms are submitted), I think this is something that would benefit from an end-to-end test.

Reply:
Okay I’ll add a check for the value tomorrow.